cluster/gce/trusty/configure-helper.sh

```diff
@@ -182,6 +184,16 @@ assemble_kubelet_flags() {
   echo "KUBELET_OPTS=\"${KUBELET_CMD_FLAGS}\"" > /etc/default/kubelet
 }
 
+start_kubelet(){
```

Comment:
Dumb bash question - is there a reason that this doesn't have the function keyword in front of it? And where is it called from?